persistence/lib/persistence/private/file_database.rb

```diff
@@ -84,7 +84,7 @@ def relative_symlinks
 
       def convert_absolute_paths_to_relative_paths(absolute_paths)
         absolute_paths.map do |file_path|
-          file_path.sub(absolute_path_in_database(separator), "")
+          file_path.sub(Regexp.new(absolute_path_in_database(separator), Regexp::IGNORECASE), "")
```

Comment:
please abstract the Regexp.new(...) into an intention-revealing method name. Thanks!